.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  vvgo:
+    runs-on: ubuntu-latest
+    run: |
+      IMAGE_ID=docker.pkg.github.com/${{ github.repository }}/$IMAGE_NAME
+
+      # Change all uppercase to lowercase
+      IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+
+      # Strip git ref prefix from version
+      VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+
+      # Strip "v" prefix from tag name
+      [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
+
+      # Use Docker `latest` tag convention
+      [ "$VERSION" == "master" ] && VERSION=latest
+
+      # Install flyctl and deploy
+      curl https://get.fly.io/flyctl.sh | sh
+      flyctl --app vvgo deploy -i $IMAGE_ID:$VERSION
```

Comment:
We need to add some tokens for this to work. Not sure how to trigger these manually to test